LoopKitUI/Views/LevelHUDView.swift

```diff
@@ -34,7 +34,7 @@ open class LevelHUDView: BaseHUDView {
         case let x? where x > 0.25:
             tintColor = stateColors?.normal
         case let x? where x > 0.10:
-            tintColor = stateColors?.normal
+            tintColor = stateColors?.warning
```

Comment:
[nit] Normally I would prefer to see policy code like this in a ViewModel and under unit test.  Please consider moving it.